src/components/common/FilterGroup/index.tsx

```diff
@@ -32,20 +33,22 @@ const useStyles = makeStyles((theme: Theme) => ({
 export type FilterField = {
   name: string;
   label: string;
+  showLabel?: boolean;
   type:
     | 'multiple_selection'
     | 'single_selection'
     | 'search'
     | 'date_range'
     | 'number_range'
     | 'count_weight'
-    | 'hidden';
+    | 'hidden'
+    | 'boolean';
 };
 
 export type FilterGroupProps = {
   initialFilters: Record<string, SearchNodePayload>;
   fields: FilterField[];
-  values: FieldValuesPayload;
+  values?: FieldValuesPayload;
```

Comment:
This was only being used by `multiple_select` and `single_select` so I made it optional.